readme.md

```diff
@@ -110,7 +110,7 @@ console.log(json);
 If you are using [Deno](https://github.com/denoland/deno), import Ky from a URL. For example, using a CDN:
 
 ```js
-import ky from 'https://esm.sh/ky';
+import ky from "https://deno.land/x/ky/source/index.ts";
```

Comment:
Single-quotes

I would also prefer to keep the esm.sh example below the new Deno.land one.

Reply:
> Did you test that this works?

😬

Reply:
I don't know. I don't use Deno.

Reply:
ok thanks, it doesn't work.

Reply:
At some point, they changed the backend architecture of deno.land so that in order to publish new releases of Ky on there, we have to set up a webhook on our repository, whereas before they had some magic to watch for GitHub releases automatically.

We just need to set up this webhook and then importing from deno.land will work again.